### PR TITLE
Edit metadata/tags for new version upload

### PR DESCRIPTION
**Pull request by kamorel**
# Description

Adds the metadata/tag modal to the version upload workflow. Metadata and tags are automatically copied from the most recent version as defaults.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/store/objectStore.ts [hunk 1]
```diff
@@ -193,10 +193,31 @@ export const useObjectStore = defineStore('object', () => {
...
 
-  async function updateObject(object: any, objectId: string, axiosOptions?: AxiosRequestConfig) {
+  async function updateObject(
+    objectId: string,
+    object: any,
+    headers: {
+      metadata?: Array<{ key: string; value: string }>,
+    },
+    params: {
+      bucketId?: string,
```
Comment (TimCsaky) [inline frontend/src/store/objectStore.ts:203]: not sure if you need bucketId (?)

**Review by TimCsaky — COMMENTED** [review]
LGTM.
the UX was awesome as well.

File: frontend/src/store/objectStore.ts (hunk 1 above)
Comment (kamorel) [reply frontend/src/store/objectStore.ts]: Correct. Removed it.

**Review by jujaga — APPROVED** [review]

**Review by TimCsaky — APPROVED** [review]

**Comment by jujaga** [comment]
Resolves #94

**Pull request merged**